lsp/src/dune

```diff
@@ -4,8 +4,6 @@
  (name lsp)
  (public_name lsp)
  (libraries jsonrpc ppx_yojson_conv_lib uutf yojson)
- (lint
-  (pps ppx_yojson_conv))
```

Comment:
We need this in development to promote the generated ppx derivers

Reply:
Should we list that dependency for development?

Reply:
I think that would be correct

Reply:
Fixed.